### ISSUE TITLE
GUI Support for Importing Submodels in UVL

### ISSUE THREAD

**Issue by SundermannC** [FeatureIDE/FeatureIDE#1250]
### Issue description
Currently, submodels can only be imported via the source editor.
It would be great to have actions in the feature model editor for:
- [ ] Adding imports
- [ ] Attach features from submodels

When importing a model it should be possible 1) select a submodel via explorer and 2) to declare an alias.
When attaching a feature from a submodel, the user should be able to select between the root features of all imported submodels.

Note, that these action should only be available for formats that support such a mechanism (probably only UVL for now).

**Comment by johannesherschel** [comment]
Yes, I am still working on it. There are still a few remaining issues with it which I am currently resolving, but it should be completed within the next few days.
